pkg/source/client/offsets_test.go

```diff
@@ -138,7 +138,7 @@ func TestInitOffsets(t *testing.T) {
 			}
 			defer sc.Close()
 			ctx := logtesting.TestContextWithLogger(t)
-			err = InitOffsets(ctx, sc, tc.topics, group)
+			_, err = InitOffsets(ctx, sc, tc.topics, group)
```

Comment:
can you assert the returned number of partitions?

Reply:
what do you mean assert the returned number of partitions?

Reply:
instead of ignoring the returned total number of partitions, just check against the expected value.